pagination/header.go

```diff
@@ -0,0 +1,49 @@
+package pagination
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+func header(u *url.URL, rel string, limit, offset int) string {
+	q := u.Query()
+	q.Set("limit", strconv.Itoa(limit))
+	q.Set("offset", strconv.Itoa(offset))
+	u.RawQuery = q.Encode()
+	return fmt.Sprintf("<%s>; rel=\"%s\"", u.String(), rel)
+}
+
+func Header(u *url.URL, count int, limit, offset int) http.Header {
```

Comment:
Maybe rename count to total? What would happen if `count` is zero (e.g. because we did not count all objects in the database)?

Reply:
I think we could either return an error or just not set `last`. We should probably choose to not set `last` as counting the total number of rows can be a pretty costly operation.

Reply:
I've renamed count to total, and if total is 0, then no `last` page is provided.